src/client/writer.rs

```diff
@@ -112,7 +112,9 @@ impl HttpClientWriter {
 
             // status line
             let _ = write!(buffer, "{} {} {:?}\r\n",
-                           msg.method(), msg.uri().path(), msg.version());
+                           msg.method(), 
+                           msg.uri().path_and_query().map(|u| u.as_str()).unwrap_or(""), 
```

Comment:
should we use .unwrap_or("/") instead of .unwrap_or("")  ?

Reply:
I was trying to match the functionality here https://docs.rs/http/0.1.5/src/http/uri/mod.rs.html#339 IE before we called path and it returned "" if there was no path.

Reply:
i think empty string is a bug

Reply:
I could see how it could be a problem. I'll return "/"